<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 60, 60);
font: 20pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>时间</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>30.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>72</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 60, 60);
font: 20pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>得分</string>
    </property>
   </widget>
   <widget class="Mouse" name="ptn0">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Mouse" name="ptn1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Mouse" name="ptn2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Mouse" name="ptn4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Mouse" name="ptn5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Mouse" name="ptn3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Mouse" name="ptn7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Mouse" name="ptn6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Mouse" name="ptn8">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>游戏</string>
    </property>
    <addaction name="actionstart"/>
    <addaction name="actionstop"/>
    <addaction name="separator"/>
    <addaction name="actionreset"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>难度设置</string>
     </property>
     <addaction name="actionfmode"/>
     <addaction name="actionmmode"/>
     <addaction name="actionhmode"/>
     <addaction name="actionhentaimode"/>
    </widget>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>音乐设置</string>
     </property>
     <addaction name="music1"/>
     <addaction name="music2"/>
    </widget>
    <addaction name="menu_4"/>
    <addaction name="menu_5"/>
    <addaction name="musicNext"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionaboutgame"/>
    <addaction name="action_QT"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionstart"/>
   <addaction name="actionstop"/>
   <addaction name="separator"/>
   <addaction name="actionexit"/>
   <addaction name="actionreset"/>
   <addaction name="separator"/>
   <addaction name="actionaboutgame"/>
   <addaction name="action_QT"/>
   <addaction name="separator"/>
   <addaction name="actionfmode"/>
   <addaction name="actionmmode"/>
   <addaction name="actionhmode"/>
   <addaction name="actionhentaimode"/>
   <addaction name="separator"/>
   <addaction name="music1"/>
   <addaction name="music2"/>
   <addaction name="musicNext"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionstart">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/images/start.png</normaloff>:/new/images/start.png</iconset>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </action>
  <action name="actionstop">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/images/stop.png</normaloff>:/new/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>结束</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/images/nonstop.png</normaloff>:/new/images/nonstop.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionreset">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/images/Refresh.png</normaloff>:/new/images/Refresh.png</iconset>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </action>
  <action name="actionaboutgame">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/images/bt02.png</normaloff>:/new/images/bt02.png</iconset>
   </property>
   <property name="text">
    <string>关于游戏</string>
   </property>
  </action>
  <action name="action_QT">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/images/qt1.png</normaloff>:/new/images/qt1.png</iconset>
   </property>
   <property name="text">
    <string>关于QT</string>
   </property>
  </action>
  <action name="actionfmode">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/images/lvl01.png</normaloff>:/new/images/lvl01.png</iconset>
   </property>
   <property name="text">
    <string>初级模式</string>
   </property>
  </action>
  <action name="actionmmode">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/images/lvl18.png</normaloff>:/new/images/lvl18.png</iconset>
   </property>
   <property name="text">
    <string>普通模式</string>
   </property>
  </action>
  <action name="actionhmode">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/images/lvl21.png</normaloff>:/new/images/lvl21.png</iconset>
   </property>
   <property name="text">
    <string>困难模式</string>
   </property>
  </action>
  <action name="actionhentaimode">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/images/lvl04.png</normaloff>:/new/images/lvl04.png</iconset>
   </property>
   <property name="text">
    <string>极难模式</string>
   </property>
  </action>
  <action name="music1">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/images/539e9c964135d.jpg</normaloff>:/new/images/539e9c964135d.jpg</iconset>
   </property>
   <property name="text">
    <string>音乐1</string>
   </property>
  </action>
  <action name="music2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/images/539e9c964135d.jpg</normaloff>:/new/images/539e9c964135d.jpg</iconset>
   </property>
   <property name="text">
    <string>音乐2</string>
   </property>
  </action>
  <action name="musicNext">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/images/iTunes.icns</normaloff>:/new/images/iTunes.icns</iconset>
   </property>
   <property name="text">
    <string>下一首</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Mouse</class>
   <extends>QPushButton</extends>
   <header>mouse.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
